packages/types/lib/keystore/index.ts

```diff
@@ -0,0 +1,16 @@
+export const entityTypes = ['session', 'connection', 'environment'] as const;
```

Comment:
Cant' be in types sorry. You can replicate the same inverted with type-fest

```ts
import type { UnionToTuple } from 'type-fest';

export type EntityType = 'session' | 'connection' | 'environment';

export const entityTypes: UnionToTuple<EntityType> = ['session', 'connection'];
// Will error, enforce all entries to be set once
````

Reply:
available in >4.26

Reply:
I actually don't need the entityTypes const. will remove